<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoPojazduObcego</class>
 <widget class="QDialog" name="OknoPojazduObcego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Obcy pojazd</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ikony/computer.png</normaloff>:/ikony/computer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="pojazdyGroupBox">
     <property name="title">
      <string>Specyfikacja pojazdu</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="pojazdyComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pociskiGroupBox">
     <property name="title">
      <string>Pocisk pojazdu</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="pociskiComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pojazdyComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OknoPojazduObcego</receiver>
   <slot>ustawPojazd(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pociskiComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OknoPojazduObcego</receiver>
   <slot>ustawPocisk(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ustawPojazd(int)</slot>
  <slot>ustawPocisk(int)</slot>
 </slots>
</ui>
